apps/website/.vuepress/theme/global-components/ClrColorListSet.vue

```diff
@@ -1,24 +1,24 @@
 <template>
   <div>
-    <div class="clr-form-control">
-      <label class="clr-control-label">Preview colors in: </label>
-      <div class="clr-control-container clr-control-inline">
-        <div class="clr-radio-wrapper">
-          <input type="radio" :id="'hsl-radio-' + listSetId" value="hsl" class="clr-radio" v-model="picked" />
-          <label :for="'hsl-radio-' + listSetId" class="clr-control-label">HSL</label>
-        </div>
-        <div class="clr-radio-wrapper">
+    <cds-form-group layout="horizontal-inline" cds-layout="m-t:md">
+      <cds-radio-group layout="horizontal-inline">
+        <label>Preview colors in:</label>
```

Comment:
nit: When setting the layout on a form group it will auto set all inputs to that layout so `cds-radio-group` wouldnt need to define a layout. Or optionally you can remove the `cds-form-group` all together instead since there is only one radio input group.